extensions/wikia/AdEngine/js/GptHelper.js

```diff
@@ -73,6 +73,23 @@ define('ext.wikia.adEngine.gptHelper', [
 		}
 	}
 
+	function setSlotLevelParams(slot, slotTargeting) {
+		var name,
+			value;
+
+		log(['setSlotLevelParams', slotTargeting], 'debug', logGroup);
+
+		for (name in slotTargeting) {
```

Comment:
Let's start with clearTargeting so we don't need to clear the amznslots manually https://developers.google.com/doubleclick-gpt/reference#googletag.Slot_clearTargeting